test/spec/automatic/spec-helper.js

```diff
@@ -37,4 +37,56 @@ includeContext = function(name, ...args) {
   return sharedExamples(name).apply(this, args);
 };
 
+let expectedImageTagFromParams = function (
```

Comment:
I'm not sure how I feel about the name `expected` - it gives the impression that it's related to `expect()` and assertion in any way.

isn't this just a regular "create" function? 
`createImageTagFromParams`?